core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1497,6 +1509,9 @@ class KafkaConfig(val props: java.util.Map[_, _], doLog: Boolean, dynamicConfigO
   var brokerId: Int = getInt(KafkaConfig.BrokerIdProp)
   val nodeId: Int = getInt(KafkaConfig.NodeIdProp)
   val processRoles: Set[ProcessRole] = parseProcessRoles()
+  val initialRegistrationTimeoutMs = getInt(KafkaConfig.InitialBrokerRegistrationTimeoutMs)
+  val brokerHeartbeatIntervalMs = getInt(KafkaConfig.BrokerHeartbeatIntervalMsProp)
+  val brokerSessionTimeoutMs = getInt(KafkaConfig.BrokerSessionTimeoutMsProp)
```

Comment:
It's worth including the explicit type here since it's a public `val`. Are these meant to be nullable? It may be better to make then `Option[Int]` if so. Otherwise, we should make them `Int` so that they're easier to use (they're currently being inferred as `Integer`).

Reply:
I added the explicit type.

I don't think they need to be nullable... they have reasonable defaults as usual in KafkaConfig.  And they will be ignored altogether when in ZK mode, of course.